docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,66 @@
+Release
+*******
+
+A release of Iris is a tag on github in the SciTools organisation master.
+
+
+Release Branch
+==============
+
+Once the features intended for the release are on master, a release branch should be created, in the SciTools/iris repository.  This will have the name:
+
+    :literal:`{major release number}.x`
+
+for example:
+
+    :literal:`v1.9.x`
+
+This branch shall be used to finalise the release details in preparation for the release candidate.
+
+Release Candidate
+=================
+
+Prior to a release, a release candidate tag shall be created, marked as a pre-release in github, with a tag ending with :literal:`rc` followed by a number, e.g.:
+
+    :literal:`v1.9.0rc1`
```

Comment:
According to the SemVer strategy, this example should ideally read "v1.9.0-rc.1".
See : [SemVer](https://github.com/mojombo/semver/blob/05e793243bcab8275e61d6d84b2d742c737593e8/semver.md) -- section 9.
I got this wrong myself with 1.9.0rc1, and only realised afterwards.
@bjlittle took the correct line, when he tagged "1.8.0-rc.1" ([here](https://github.com/SciTools/iris/commit/1b24f73b32bccd0c1c8ca756e5c9cc3cde336c27#diff-0f7e8886af1cb241f78fb43f343d5351)).